hw/dv/tools/dvsim/testplans/shadow_reg_errors_testplan.hjson

```diff
@@ -7,21 +7,18 @@
       // this testplan should be imported by all IPs containing shadowed CSRs
       name: shadow_reg_errors
       desc: '''
-            Verify shadow registers' update and storage errors.
-            - Issue reset at random to clear all the internal stored values and phases trackers
-              in shadow registers.
-            - Select all the shadow registers and a random amount of the rest of the non-excluded
-              registers. Shuffle and write random values to the selected registers.
-              For shadow registers, the second write is not enabled.
-              There is a 50% possibility that the shadow register's write value is identical to its
-              previous write value.
-              If shadow register's second write value does not match the first write value,
-              ensure that the update error alert is triggered.
-            - Randomly inject storage errors by modifying the shadow register's staged or committed
-              values via backdoor method. Ensure that the storage error alert is triggered.
-            - Randomly decide to read all the non-excluded registers or fields. Then check the read
-              values against predicted values.
-              A read on a shadow register will clear its phase tracker.
+            Verify shadowed registers' update and storage errors.
+            - Randomly select a shadowed register.
+            - Write two times to the shadowed register with different values.
+              Check if the update error alert is trigger and if shadowed register value is
+              unchanged.
+            - Write once to the shadowed register and read to clear the staged value.
+              Then write twice with the same value to ensure the write is successful and no update
+              error alert.
+            - Backdoor write to shadowed or committed flops to create a storage fatal alert.
+              Check if fatal alert continuously fires until reset.
+              Check if any other frontdoor write attempt is blocked during the storage error.
+            - Reset the DUT.
```

Comment:
```suggestion
     desc: '''Verify shadowed registers' update and storage errors.
            
            For all shadow registers in the DUT:
            - Randomly pick a shadowed register.
            - Write it twice with different values, verify that the update error alert is triggered
              and the register value remains unchanged.
            - Write it once and read it back to clear the staged value.
            - Then write it twice with the same new value (but different from the previous step).
            - Read it back to verify the new value and ensure that the update error alert did not
              trigger.
            - Backdoor write to shadowed or committed flops to create a storage fatal alert.
            - Check if fatal alert continuously fires until reset.
            - Verify that all other frontdoor write attempts are blocked during the storage error.
            - Reset the DUT.
            - Read all CSRs to ensure the DUT is properly reset. 
```

Reply:
Updated, thanks Sri for the detailed suggestion.